### PR TITLE
feat(mail): create email sending

### PR DESCRIPTION
**Pull request by hmartiins**
### **Proposed changes**

Create email sending using Amazon SES.

<br>

### **Additional Information**

Creating the sending of email using Amazon SES, I implemented this feature only in SendForgotPasswordService, if you have any other place that needs to send email please say. And another important information, at the moment, the sending of email works only for the domains 'peruibetec@gmail.com' or 'suporte@peruibetec.com', since the Amazon SES environment is still configured in SandBox mode, which would be the testing mode , when the application is for production, I will take it out of SandBox mode and allow sending emails to all domains

### CODE REVIEW

**Comment by devLeSilverio** [comment]
LGTM !!

File: src/modules/user/view/forgot_password.hbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<div class="message-content">
+  <p>Olá, {{name}}</p>
+  <p>Parece que uma troca de senha para sua conta foi solicitada</p>
+
+  <p>Se foi você, então clique no link abaixo para escolher uma nova senha:</p>
+  <p>
+    <a href="{{link}}">Resetar minha senha</a>
+  </p>
+
```
Comment (devLeSilverio) [inline src/modules/user/view/forgot_password.hbs]: I think it is valid for user accessibility to add:

Está com problemas para acessar o link acima? Use este link:
{{link}} 

Para sua segurança, este link irá expirar em x horas ou imediatamente após você redefinir sua senha.

**Review by devLeSilverio — CHANGES_REQUESTED** [review]
I found no problem, just left a suggestion! LGTM.

File: src/modules/user/view/forgot_password.hbs (hunk 1 above)
Comment (hmartiins) [reply src/modules/user/view/forgot_password.hbs]: done !

**Review by devLeSilverio — APPROVED** [review]
LGTM!

**Review by joaovds — CHANGES_REQUESTED** [review]
Just a note here, update the .env.example file with these new variables for this new feature. despite that, LGTM!

**Review by joaovds — APPROVED** [review]

**Review by joaovds — APPROVED** [review]
LGTM!

**Pull request merged**